e2e-tests/adapters/netlify.toml

```diff
@@ -1,3 +1,9 @@
 [build]
   command = "npm run build"
-  publish = "public/"
\ No newline at end of file
+  publish = "public/"
+
+[build.environment]
+  NETLIFY_IMAGE_CDN = "true"
+
+[images]
+  remote_images = ["https://images.unsplash.com/*"]
```

Comment:
Should we update this to the "correct" format?

Reply:
Done in https://github.com/gatsbyjs/gatsby/pull/38685/commits/1bb2e01b88bf352c6fe6c8f81ca6e27bcb934df2